src/segment-loader.js

```diff
@@ -413,6 +413,11 @@ export const shouldFixBadTimelineChanges = (timelineChangeController) => {
   const pendingAudioTimelineChange = timelineChangeController.pendingTimelineChange({ type: 'audio' });
   const pendingMainTimelineChange = timelineChangeController.pendingTimelineChange({ type: 'main' });
   const hasPendingTimelineChanges = pendingAudioTimelineChange && pendingMainTimelineChange;
+
+  if (hasPendingTimelineChanges && pendingAudioTimelineChange.to < pendingMainTimelineChange.to) {
+    timelineChangeController.trigger('audioTimelineBehind');
```

Comment:
Should it be inside `should fix bad timeline changes`?
this looks like a side effect, and should be moved outside

Reply:
I agree that this can be moved elsewhere.. but I think there are a few different places it could be placed.

Do you think it would make sense to just check after an [audio timeline change](https://github.com/videojs/http-streaming/blob/main/src/segment-loader.js#L723)?

Reply:
probably yes